server/deskroom/knowledge_base/endpoints.py

```diff
@@ -115,6 +117,7 @@ async def update_knowledge_base(
                     "frequently_asked": item.frequently_asked,
                     "caution_required": item.caution_required,
                     "category_id": item.category_id,
+                    "updated_at": datetime.now(timezone.utc).isoformat(),
```

Comment:
이거 pydantic을 좀 더 잘 쓰면 해결 할 수 있지 않을까요? 


```suggestion
await supabase.table("knowledge_base")
            .update(item.dict(exclude_unset=True)).eq("id", item.id)
```
요렇게 하고, 
KnowledgeBaseUpdateIn에서 
```
class KnowledgeBaseUpdateIn(Schema):
    ...
    updated_at: datetime = Field(default_factory=datetime.now)

    # json field에서 isoformat으로 바꾸게
```
이러면 위 코드가 훨씬 간결해질 거 같습니다. 매번 반구조화를 하는 건 DTO의 역할을 충분히 다해내지 못하는 게 아닐까요?

Reply:
생각해보니 이 경우는 updated_at이 자동으로 업데이트 안되서 그런거 같네요. 이 경우 trigger를 써서 update 바로 되도록 할 수 있으니 그 방향으로 가는 게 어떨까요? 

Reference: https://www.reddit.com/r/PostgreSQL/comments/10shxej/comment/j71ggl8/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

Reply:
저도 트리거 고민해봤었는데.. 코드가 좀 더 명시적인거 같아서 위처럼 코드에 반영해보았었습니다.
근데 다시보니 코드에서 처리하는게 좀 이상하긴 하네요.
나중에 Supabase 벗어날 때, 컬럼 제약 조건으로 추가하는 방식을 고려해봐야겠네요.